content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,129 +2,169 @@
-box next to :guilabel:`Automatic Assignment` to enable the feature for this team.
+When tickets are received, they need to be assigned to a member of the team. This is done either
+manually on each ticket individually, or through :guilabel:`Automatic Assignment`. Check the
+:guilabel:`Automatic Assignment` checkbox to enable this feature for the team.
 
-.. image:: getting_started/assignment-visibility.png
+.. image:: getting_started/helpdesk-visibility-assignment.png
    :align: center
-   :alt: View of a Helpdesk team settings page emphasizing the assignment and visibility features
-         in Odoo Helpdesk
+   :alt: View of a Helpdesk team settings page emphasizing the automatic assignment features in Odoo
+         Helpdesk.
 
 Select one of the following assignment methods, based on how workload should be allocated across
 the team:
 
-- :guilabel:`Each user is assigned an equal number of tickets` assigns tickets to team members
-   based on total ticket count, regardless of the number of open or closed tickets they are
-   currently assigned.
-- :guilabel:`Each user has an equal number of open tickets` assigned tickets to team members based
-   on how many open tickets they are currently assigned. This option is useful for automatically
-   delegating a heavier workload to high-performers who tend to close tickets quickly.
+- :guilabel:`Each user is assigned an equal number of tickets`: tickets are assigned to team members
+  based on total ticket count, regardless of the number of open or closed tickets they are
+  currently assigned.
+- :guilabel:`Each user has an equal number of open tickets`: tickets are assigned to team members
+  based on how many open tickets they are currently assigned.
+
+.. note::
+   When :guilabel:`Each user is assigned an equal number of tickets` is selected, the overall number
+   of tickets assigned to team members is the same, but it does not consider the current workload.
+   When :guilabel:`Each user has an equal number of open tickets` is selected, it ensures a balanced
+   workload among team members, as it takes the current number of active tickets into account.
 
-Finally, add the :guilabel:`Team Members` who will be assigned tickets for this team. Leave the
+Finally, add the :guilabel:`Team Members` who are to be assigned tickets for this team. Leave the
 field empty to include all employees who have the proper assignments and access rights configured in
 their user account settings.
 
-.. note::
-   If an employee has time off scheduled in the :guilabel:`Time Off` application, they will not be
-   assigned tickets during that time. If no employees are available, the system will look ahead
-   until there is a match.
+.. important::
+   If an employee has time off scheduled in the *Time Off* application, they are not assigned
+   tickets during that time. If no employees are available, the system looks ahead until there is a
+   match.
```

Comment:
The problem with including all of the other fields, is it would essentially cover all of the other docs, which would make this one document at least a thousand lines by my estimate. I know that in some cases having a setup/configuration document that length makes sense, I don't think it does in this case. Also, once the content for these docs has been updated for 17, I have plans to rework the structure (hopefully like the Accounting docs, using the card format) to clarify what document covers what section of the form. I have a draft that I can show if you if you are interested, I stopped working on it to get the content up to date before I started messing with the structure too much.